CHANGELOG.md

```diff
@@ -6,6 +6,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [22.1.0]
+### Added
+- Add method to check and update collectible ownership state ([#664](https://github.com/MetaMask/controllers/pull/664))
+- Add finish to EIP-1559 v2 gas estimate code ([#660](https://github.com/MetaMask/controllers/pull/660))
+- [FEAT] Favorite Collectibles ([#623](https://github.com/MetaMask/controllers/pull/623))
+- Use eth_feeHistory as a fallback for gas estimates ([#614](https://github.com/MetaMask/controllers/pull/614))
+- Pull in remaining EIP-1559 v2 datapoints ([#646](https://github.com/MetaMask/controllers/pull/646))
+- Pull in networkCongestion along with gas estimates ([#632](https://github.com/MetaMask/controllers/pull/632))
+- Extract determineGasFeeCalculations ([#612](https://github.com/MetaMask/controllers/pull/612))
```

Comment:
What do you think if we left out #612 (since it's just a refactoring commit) and said:

``` suggestion
- Update GasFeeController to use `eth_feeHistory` to compute gas fee recommendations when MetaSwap API is down ([#614](https://github.com/MetaMask/controllers/pull/614))
- Update GasFeeController to expose additional data sourced from the MetaSwap API in support of design updates to "edit gas fee" functionality in extension ([#632](https://github.com/MetaMask/controllers/pull/632)), ([#646](https://github.com/MetaMask/controllers/pull/646))
```